src/geometry/euclidean/circumcenter.lean

```diff
@@ -583,3 +619,77 @@ end
 end simplex
 
 end affine
+
+namespace euclidean_geometry
+
+open affine affine_subspace
+
+variables {V : Type*} {P : Type*} [inner_product_space V] [metric_space P]
+    [normed_add_torsor V P]
+include V
+
+/-- Suppose all distances from `p₁` and `p₂` to the points of a
+simplex are equal, and that `p₁` and `p₂` lie in the affine span of
+`p` with the vertices of that simplex.  Then `p₁` and `p₂` are equal
+or reflections of each other in the affine span of the vertices of the
+simplex. -/
+lemma eq_or_eq_reflection_of_dist_eq {n : ℕ} {s : simplex ℝ P n} {p p₁ p₂ : P} {r : ℝ}
+    (hp₁ : p₁ ∈ affine_span ℝ (insert p (set.range s.points)))
+    (hp₂ : p₂ ∈ affine_span ℝ (insert p (set.range s.points)))
+    (h₁ : ∀ i, dist (s.points i) p₁ = r) (h₂ : ∀ i, dist (s.points i) p₂ = r) :
+  p₁ = p₂ ∨ p₁ = reflection (affine_span ℝ (set.range s.points)) p₂ :=
+begin
+  have h₁' := s.orthogonal_projection_eq_circumcenter_of_dist_eq' h₁,
```

Comment:
I've said this before... I think it would help if you make a follow-up PR that renames all `orthogonal_projection` substring to `ortho_proj`. I hope that it will reduce the number of line-wraps and increase the overall readability.